crowdfunding.config.json

```diff
@@ -0,0 +1,4 @@
+{
+  "address": "",
```

Comment:
I suppose this is related with "Since I am using a local version of the contract, it makes no sense to set the address and the abi of the contract config file." 

I don't understand the idea, ABI should be the same for local contract and a Testnet deployed one. 

Also, youre using this file on `index.page.js` so it will probably not work with empty values right?

Reply:
Exactly, it won't work. You are right, the ABI will be the same, but the address will change, right?

Reply:
Yes, it could change. What about using env variables? Like taking the address from process.env.CROWDFUNDING_ADDRESS?